sections/faqs/dom-two-classes.js

```diff
@@ -0,0 +1,13 @@
+import md from 'components/md'
+
+const TwoDomClasses = () => md`
+  ## Why does a DOM node have two classes?
+
+  Each node actually has two classes connected to it;
+
+  One is static per component meaning that each instance of a component has this class, but it hasn't any style attached to it. However, it can be used to help the developer smoothly tweak all instances of a class at once.
```

Comment:
> per *styled* component

> component*,* meaning

> each ~instance~ element of ~a~ *the styled component*.

> . But it *doesn't have* any style*s* attached to it.

Reply:
> However, it can be used to help the developer smoothly tweak all instances of a class at once

This is not the actual intent. It is used to quickly **identify** the component, and to be able to make some quick changes in the DevTools.

Reply:
Alright, I guess I misunderstood a little bit. But speaking as a "noob", I would not understand what's meant by "it identifies the component quickly". 

Would you mind elaborating because I still don't understand/know what to write here.

Reply:
@nilsgoksor identify, as in, looking at the DOM element and knowing due to the class which ones come from the same styled component